github/apps.go

```diff
@@ -62,6 +62,12 @@ type Installation struct {
 	UpdatedAt           *Timestamp               `json:"updated_at,omitempty"`
 }
 
+type Attachment struct {
```

Comment:
Please document this new struct similarly to how the other structs in this repo are documented.